### PR TITLE
Describe hooks scoping possibility in the hooks doc description part

### PR DESCRIPTION
**Pull request by yamalight**
As discussed in https://github.com/fastify/fastify/issues/487

### CODE REVIEW

File: docs/Hooks.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Hooks
 
-By using the hooks you can interact directly inside the lifecycle of Fastify, there are three different Hooks that you can use *(in order of execution)*:
+By using the hooks you can interact directly inside the lifecycle of Fastify, there are five different Hooks that you can use *(in order of execution)*:
```
Comment (jsumners) [inline docs/Hooks.md]: > ... Fastify. There are...

File: docs/Hooks.md [hunk 2]
```diff
@@ -82,7 +82,8 @@ fastify.addHook('onResponse', async (res) => {
 | reply | Fastify [Reply](https://github.com/fastify/fastify/blob/master/docs/Reply.md) interface |
 | next | Function to continue with the [lifecycle](https://github.com/fastify/fastify/blob/master/docs/Lifecycle.md) |
 
-It is pretty easy to understand where each hook is executed, if you need a visual feedback take a look to the [lifecycle](https://github.com/fastify/fastify/blob/master/docs/Lifecycle.md) page.
+It is pretty easy to understand where each hook is executed, if you need a visual feedback take a look to the [lifecycle](https://github.com/fastify/fastify/blob/master/docs/Lifecycle.md) page.
```
Comment (jsumners) [inline docs/Hooks.md]: > It is pretty easy to understand where each hook is educated by looking at the [lifecycle page](https://github.com/fastify/fastify/blob/master/docs/Lifecycle.md).

File: docs/Hooks.md [hunk 3]
```diff
@@ -82,7 +82,8 @@ fastify.addHook('onResponse', async (res) => {
 | reply | Fastify [Reply](https://github.com/fastify/fastify/blob/master/docs/Reply.md) interface |
 | next | Function to continue with the [lifecycle](https://github.com/fastify/fastify/blob/master/docs/Lifecycle.md) |
 
-It is pretty easy to understand where each hook is executed, if you need a visual feedback take a look to the [lifecycle](https://github.com/fastify/fastify/blob/master/docs/Lifecycle.md) page.
+It is pretty easy to understand where each hook is executed, if you need a visual feedback take a look to the [lifecycle](https://github.com/fastify/fastify/blob/master/docs/Lifecycle.md) page.  
+Hooks also support scopes that can be used to apply hooks to selected routes only, see [Scopes](#scope) section for more info.
```
Comment (jsumners) [inline docs/Hooks.md]: > Hooks are affected by Fastify's encapsulation, and can thus be applied to selected routes. See the [Scopes](#scope) section for more information.

**Comment by yamalight** [comment]
@jsumners updated!

File: docs/Hooks.md [hunk 4]
```diff
@@ -82,7 +82,8 @@ fastify.addHook('onResponse', async (res) => {
 | reply | Fastify [Reply](https://github.com/fastify/fastify/blob/master/docs/Reply.md) interface |
 | next | Function to continue with the [lifecycle](https://github.com/fastify/fastify/blob/master/docs/Lifecycle.md) |
 
-It is pretty easy to understand where each hook is executed, if you need a visual feedback take a look to the [lifecycle](https://github.com/fastify/fastify/blob/master/docs/Lifecycle.md) page.
+It is pretty easy to understand where each hook is educated by looking at the [lifecycle page](https://github.com/fastify/fastify/blob/master/docs/Lifecycle.md).
```
Comment (jsumners) [inline docs/Hooks.md]: Hooks don't receive an education 😆

Totally my fault in my last comment. I have no idea where that word came from. I should still be "executed".

**Comment by yamalight** [comment]
@jsumners whoops, not sure how I overlooked that. Fixed 😄

**Pull request merged**